### PR TITLE
TRUNK-295: Removed addProxyPrivilege and removeProxyPrivilege methods

### PR DESCRIPTION
**Pull request by SsebaddukaG**
https://issues.openmrs.org/browse/TRUNK-295

Remove Context.addProxyPrivilege
TRUNK-295

Removed the addProxyPrivilege and removeProxyPrivilege methods.

### CODE REVIEW

**Comment by SsebaddukaG** [comment]
Still having a failure with the coverage build could someone advise me on what might be the cause because am a successful build on my IDE locally.

**Comment by dkayiwa** [comment]
After making these changes, did you try run them with the reference application?

**Comment by SsebaddukaG** [comment]
I actually didn’t. I just did a maven clean package which was a success. Is there a reference to direct me on how I can with the reference application? It’s something I wasn’t familiar with

**Comment by dkayiwa** [comment]
Can you ask on talk?

**Comment by SsebaddukaG** [comment]
Okay.. thank you?

**Comment by SsebaddukaG** [comment]
I have run with the reference Application and it works fine.

File: api/src/main/java/org/openmrs/api/OrderService.java [hunk 1]
```diff
@@ -323,7 +323,7 @@
 	 * 
 	 * @return the order number seed
 	 */
-	@Authorized(PrivilegeConstants.ADD_ORDERS)
+	//@Authorized(PrivilegeConstants.ADD_ORDERS)
```
Comment (ODORA0) [inline api/src/main/java/org/openmrs/api/OrderService.java]: @SsebaddukaG Shouldn't you remove this than have this commented out?

**Comment by sherrif10** [comment]
@dkayiwa  following the above comments , is there any way of how i can test my code in done  openmrs-core after writing  all unit tests  to be able to see changes in reference application. any help here

**Comment by sherrif10** [comment]
have been following this link https://talk.openmrs.org/t/how-to-see-effects-of-code-change-on-the-refapp/22879/2 but it only talks about LUI code

**Comment by dkayiwa** [comment]
Can you ask the wider audience on OpenMRS Talk?

**Pull request closed without merging**